### DISCUSSION TITLE
Great app!

### DISCUSSION THREAD

**Discussion by robcopter** [unknownskl/xbox-xcloud-client#413] [General]
Just wanted to give a huge props for a great app! Works flawless on my m1 mac. 
Keep up the great work!

**Comment by diggoran** [comment]
I originally found this app while searching for ways to lower the resolution of my stream in the first party Xbox app, when connecting to my Xbox Series X on my home network via an ethernet connection.

I was blown away by how perfect the connection is after using this app. It's streaming at full resolution, full framerate, with no dips, drops, or lags at all. I don't understand how Microsoft could make such a poor streaming app that I was doubting the capability of my current-gen hardware over a wired connection.

And this doesn't just improve the local streaming, but also the cloud streaming too! A much better experience all around!

Microsoft, hire this dev and integrate these improvements before you cancel your xCloud service due to shooting yourself in the foot for years!

**Reply by carterlasalle** [reply]
@diggoran I am really struggling to emulate this result.

**Comment by marcobag** [comment]
I agree, thank you so much this app is wonderful! I would like to donate a small amount to support the project.
Is there any donation link?

**Comment by OptimusGREEN** [comment]
Yup, thanks so much for this.
